docs/routing/route_configuration_options.md

```diff
@@ -2,86 +2,108 @@
 
 ## Route domains
 
-ILC will check the exact match of domain names, accordingly, `secure.example.com` is not equal to `example.com`, so you may need to add the necessary subdomains.
+ILC checks the exact match of domain names. It means that `secure.example.com` is not equal to `example.com`, and you may need to add the necessary subdomains to handle this case properly.
 
-ILC can handle requests coming from multiple domains, so that you don't need to roll out individual instances of the ILC for every domain, you just can use one instance for all of them.
+ILC can handle requests coming from multiple domains, so you can use a single ILC instance to handle them instead of rolling out individual instances for every domain.
 
 ![ILC registry domains field](../assets/routes/domain-field.png)
 
-- Domain name, must be without protocol, etc.
+!!! note ""
+    - Domain name must be without protocol.
+    - Empty `Domain Name` field means the main app domain.
+    - To add a new domain, go to the **Router domains** section in the sidebar.
+        
+        ??? tip "Add a new domain"
+            ![ILC registry domains menu](../assets/routes/domain-create.png)
 
-- You can add new domain in **Router domains** menu.
+!!! note ""
+    The application renders only one domain at the same time. To add one header to several domains, you need to create the same route several times specifying the required domain for each route.
```

Comment:
The application renders only one domain at the same time. 

This is unclear to me... Which application?

Reply:
The original sentence was kept here, so the information is available in the documentation as well.

<img width="1385" alt="Screen Shot 2022-02-04 at 10 41 00" src="https://user-images.githubusercontent.com/16765690/152498333-93c7d44e-c661-4afb-9a18-480f36f43636.png">

<img width="705" alt="Screen Shot 2022-02-04 at 10 43 42" src="https://user-images.githubusercontent.com/16765690/152498605-54347253-12d0-4f6f-9003-e39ca8ed0a23.png">

I believe this aspect wasn't spot during the initial review. @DammneD ^^ FYI

Reply:
Clarified with @DammneD